charts/latest/azuredisk-csi-driver/values.yaml

```diff
@@ -6,19 +6,19 @@ image:
     pullPolicy: IfNotPresent
   csiProvisioner:
     repository: /oss/kubernetes-csi/csi-provisioner
-    tag: v2.1.1
+    tag: v2.2.2
```

Comment:
These changes will need to be made to the v2.0.0-alpha.1 chart as well.

Reply:
Made the changes